main.go

```diff
@@ -146,6 +146,7 @@ func mainWithError() (err error) {
 
 	daemonCommand := newCommand.DaemonCommand().CobraCommand()
 
+	daemonCommand.PersistentFlags().String(f.Service.App.PauseAnnotation, "", "Annotation used to pause App CR reoncilliation.")
```

Comment:
```suggestion
	daemonCommand.PersistentFlags().String(f.Service.App.PauseAnnotations, "", "Annotations used to pause App CR reoncilliation.")
```

Looks like it's apparently plural, right?

Reply:
I wanted to have just 1 annotation (excluding `operatorkit` defaults) which can pause app-operator, hence singular.

If you think it would be beneficial to extend this and allow for multiple annotations, let me know. That would be a separate PR though.